src/backend/BlockchainCertificatesIssuer/BlockchainCertificatesIssuer/API/CertificateAPI.cs

```diff
@@ -28,9 +28,7 @@ public async Task<HttpResponseData> Run([HttpTrigger(AuthorizationLevel.Anonymou
             var certificate = await System.Text.Json.JsonSerializer.DeserializeAsync<Certificate>(req.Body);
 
             var response = req.CreateResponse(HttpStatusCode.OK);
-            response.Headers.Add("Content-Type", "text/plain; charset=utf-8");
-
-
+          
             var created = await repository.CreateAsync(new Certificate { Course = certificate.Course, Trainee = certificate.Trainee, Trainer = certificate.Trainer, CertificateIssueDate =certificate.CertificateIssueDate });
```

Comment:
This line can be simply written as `var created = await repository.CreateAsync(certificate);`

Reply:
Done